modules/ossm-security-constraints.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/prepare-to-deploy-applications-ossm.adoc
+
+
+[id="ossm-security-constraints_{context}"]
+= {ProductName} security constraints
+
+[NOTE]
+====
+The relaxing of security constraints is only necessary during the {ProductName} Technology Preview.
+====
+
+When you deploy an application into a {ProductShortName} running in an OpenShift environment, it is necessary to relax the security constraints placed on the application by its service account to ensure the application can function correctly. Each service account must be granted permissions with the `anyuid` and `privileged` Security Context Constraints (SCC) to enable the sidecars to run correctly.
+
+The `privileged` SCC is required to ensure changes to the pod's networking configuration is updated successfully with the `istio-init` initialization container and the `anyuid` SCC is required to enable the sidecar container to run with its required user id of `1337`.
+
+To configure the correct permissions it is necessary to identify the service accounts being used by your application's pods. For most applications, this will be the `default` service account, however your Deployment/DeploymentConfig may override this within the pod specification by providing the `serviceAccountName`.
+
+For each identified service account you must update the cluster configuration to ensure they are granted access to the `anyuid` and `privileged` SCCs by executing the following commands from an account with cluster admin privileges.
```

Comment:
Missing the commands to add permissions? These should only be required if using proxy-init and not CNI. Proxy-init is currently the default. 

`oc adm policy add-scc-to-user anyuid -z default -n bookinfo
  $ oc adm policy add-scc-to-user privileged -z default -n bookinfo`

Reply:
@brian-avery The commands are in the ossm-configure-security-constraints.adoc module. 

Are they no longer correct?

Reply:
Hmmm. What I see on GitHub is:

SCCs by executing the following commands from an account with cluster admin privileges. <END OF DOC>

Reply:
Ah nvm. I see what's going on.